Content.Shared/Movement/Components/SlowContactsComponent.cs

```diff
@@ -17,6 +17,14 @@ public sealed partial class SlowContactsComponent : Component
     [DataField("sprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
     public float SprintSpeedModifier = 1.0f;
 
+    [DataField("spiderWalkSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
+    public float SpiderWalkSpeedModifier = 1.5f;
+
+    [AutoNetworkedField]
+    [DataField("spiderSprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    public float SpiderSprintSpeedModifier = 1.5f;
+
```

Comment:
Do not hardcode in specific entities like this as it breaks composition.

If you want it whitelisted then use an entity whitelist.